app/controllers/sightings_controller.rb

```diff
@@ -0,0 +1,49 @@
+class SightingsController < ApplicationController
+    def index
+        sightings = Sighting.all
+        render json: sightings
+    end
+
+    def show 
+        sighting = Sighting.find(params[:id])
+        render json: sighting
+      end
+
+    def create
+        sighting = Sighting.create(sighting_params)
+        if sighting.valid?
+          render json: sighting
+        else
+          render json: sighting.errors
+        end
+        
+    end
+
+      def update
+        sighting = Sighting.find(params[:id])
+        sighting.update(sighting_params)
+        if sighting.valid?
+          render json: sighting
+        else
+          render json: sighting.errors
+        end
+      end
+
+      def destroy
+        sighting = Sighting.find(params[:id])
+        if sighting.destroy
+          render json: sighting
+        else
+          render json: sighting.errors
+        end
+    end
+    
+    private
+    def sighting_params
+        params.require(:sighting).permit(:longitude, :date, :latitude, :animal_tracker_id)
+      end
+end
+
+t.integer "animal_id"
+
+rails generate migration rename_column :sightings, :animal_id, :animal_tracker_id
```

Comment:
Let's remove this from this file, although I recommend adding it to wherever you are documenting your project.